### ISSUE TITLE
can not use with ubuntu 16.04 lst

### ISSUE THREAD

**Issue by arkofgalaxy** [PaddleCV-SIG/iann#105]
install the eiseg with pip ,open it said that:
![Selection_006](https://user-images.githubusercontent.com/28675720/153351978-b3d4ef5a-9224-4387-9f2b-30a0577ecb09.png)
and clone it ,it said the same.
that is the conda environment list.
[conda list.txt](https://github.com/PaddleCV-SIG/EISeg/files/8038677/conda.list.txt)

**Comment by geoyee** [comment]
please try to upgrade PyQt from 5.12 to 5.13+ and see if the problem is solved

**Comment by geoyee** [comment]
do you solve it?

**Comment by arkofgalaxy** [comment]
> do you solve it?

no,the 5.13 version can not install. T A T. I try to solve it , but not find a useful way.

**Comment by geoyee** [comment]
> > do you solve it?
> 
> no,the 5.13 version can not install. T A T. I try to solve it , but not find a useful way.

ok, i find a samed problem as your in some forums, they told me upgrade PyQt > 5.13+, because this is a old problem in PyQt and solved in 5.13.
